demisto_sdk/commands/generate_test_playbook/tests/test_playbook_generator_test.py

```diff
@@ -22,12 +22,12 @@ class TestGenerateTestPlaybook:
 
     @classmethod
     def setup_class(cls):
-        print("Setups TestGenerateTestPlaybook class")
+        print("Setups TestGenerateTestPlaybook class")  # noqa: T201
```

Comment:
can't we use a logger here?

Reply:
It's a test. We don't use the logger here.